docs/reference/system-requirements.rst

```diff
@@ -107,12 +111,11 @@ Virtualization support
 ROCm supports virtualization for select GPUs only as shown below.
 
 .. csv-table::
-    :widths: 50, 25, 25, 50
+    :widths: 25, 25, 25, 50
     :header: "Hypervisor", "Version", "GPU", "Validated guest OS (kernel)"
 
-    "VMWare", "ESXI 8", "MI250", "Ubuntu 20.04 (5.15 [HWE])"
-    "VMWare", "ESXI 8", "MI210", "Ubuntu 20.04 (5.15 [HWE]), SLES 15 SP4 (5.14.21)"
-    "VMWare", "ESXI 7", "MI210", "Ubuntu 20.04 (5.15 [HWE]), SLES 15 SP4 (5.14.21)"
+    "VMWare", "ESXI 8", "MI210", "Ubuntu 22.04.4 (6.8 [HWE]), |br| SLES 15 SP5 (5.14.21), |br| RHEL 8.10 (5.14.0)"
+    "VMWare", "ESXI 7", "MI210", "Ubuntu 22.04.4 (6.8 [HWE]), |br| RHEL 8.10 (5.14.0)"
```

Comment:
```suggestion
    "VMWare", "ESXi 8", "MI210", "Ubuntu 22.04.4 (6.8 [HWE]), |br| SLES 15 SP5 (5.14.21), |br| RHEL 8.10 (5.14.0)"
    "VMWare", "ESXi 7", "MI210", "Ubuntu 22.04.4 (6.8 [HWE]), |br| RHEL 8.10 (5.14.0)"
```

Reply:
I'm a little surprised we don't have more detail about the ESXi version numbers. 
The website shows more detail - https://knowledge.broadcom.com/external/article/316595/build-numbers-and-versions-of-vmware-esx.html
![image](https://github.com/user-attachments/assets/f0f315f4-9297-4ae7-9252-a767b00274df)

Reply:
Updated to EXSi 7.0.3 and 8.0.3